spec/manipulateFile_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'manipulateFile'
+
+describe 'ManipulateFile' do
+    it 'should read the first line of the file' do   
+        @content = "Test 1,2,3"
+
+        arq = File.new("test.log","w")
+        arq.puts @content
+        arq.close
+
+        main = ManipulateFile.new(arq)
+
+        expect(main.getFirstLine).to eq(@content)
+    end
+
+    it 'should treat error when file doesn`t exist' do
```

Comment:
```suggestion
    it 'treat error when file doesn`t exist' do
```

Uma sugestão do betterspecs aqui(https://www.betterspecs.org/#expect) é que não devemos utilizar should. As asserções devem ser objetivas.